src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Lookup.txt

```diff
@@ -119,3 +119,6 @@ Errors: Error 23-28: Name isn't valid. 'Value' isn't recognized.
 
 >> LookUp([1, 2, 3] As X, X.Value > 2, Value)
 Errors: Error 36-41: Name isn't valid. 'Value' isn't recognized.
+
+>> LookUp(Table({x:1}, Error({Kind:ErrorKind.Custom}), {x:3}), ThisRecord.x = 3)
```

Comment:
Some interesting test to try with this issue:

    >> CountIf(Filter([{a:1},If(1/0<2,{a:2}),{a:3}], IsError(a)), true)
    1
    
    >> Concat( Filter([{a:1},If(1/0<2,{a:2}),{a:3},If(Sqrt(-1)<0,{a:4}),{a:5}], IfError(a, FirstError.Kind) < 20), IfError($"A = {a}", $"A is error code {FirstError.Kind}"), "," )
    "A = 1, A is error code 13, A = 3, A = 5"
    
    >> LookUp( [{a:1},If(1/0<2,{a:2}),{a:3},If(Sqrt(-1)<0,{a:4}),{a:5}], IsError(a) && IfError(a, FirstError.Kind) > 20 )
    Error({Kind:ErrorKind.Numeric})
    
    >> IfError( LookUp( [{a:1},If(1/0<2,{a:2}),{a:3},If(Sqrt(-1)<0,{a:4}),{a:5}], IsError(a) && IfError(a, FirstError.Kind) > 20 ).a, FirstError.Kind )
    24

Reply:
Test cases added with the exception of the last one, which falls into the [https://github.com/microsoft/Power-Fx/issues/1322](https://github.com/microsoft/Power-Fx/issues/1322) issue.